app/assets/javascripts/discourse/app/routes/discovery-categories.js

```diff
@@ -78,7 +81,10 @@ const DiscoveryCategoriesRoute = DiscourseRoute.extend(OpenComposer, {
     };
   },
 
-  _findCategoriesAndTopics(filter) {
+  // the extras parameter is not used in discourse core but it is provided in case some plugins need to implement more
+  // complex loading strategies
+  // eslint-disable-next-line no-unused-vars
+  _findCategoriesAndTopics(filter, extras) {
```

Comment:
I'm a little confused with this change because this function accepts the `extras` args but it doesn't seem to be used in the function.

Reply:
You're right. This was a little fishy as this parameter was not intended to be used in core, but instead in a customer plugin that overrides this function.